yarn.lock

```diff
@@ -10386,7 +10386,7 @@ eth-json-rpc-middleware@^4.1.4, eth-json-rpc-middleware@^4.1.5, eth-json-rpc-mid
     pify "^3.0.0"
     safe-event-emitter "^1.0.1"
 
-eth-keyring-controller@^5.3.0, eth-keyring-controller@^5.5.0:
+eth-keyring-controller@^5.3.0:
   version "5.5.0"
```

Comment:
Could you update this one too? This is the one GABA is using, so it'd be ideal to keep it in sync

Reply:
Fixed in 56433c5

Reply:
That didn't seem to work - GABA is still using v5.5.0 of the keyring controller

Reply:
Hm, we should update that.